hs_core/models.py

```diff
@@ -267,6 +267,13 @@ def create(cls, **kwargs):
     def update(cls, element_id, **kwargs):
         element_name = cls.__name__
         creator_order = None
+        if 'description' in kwargs:
+            party = cls.objects.get(id=element_id)
+            if party.description is not None and kwargs['description'] is not None:
+                if len(party.description) > 0 and len(kwargs['description']) > 0:
```

Comment:
Empty string ' '  is not equal to ```None```.  For that reason both checks are needed, However, I should be using ```len(party.description.strip()) > 0``` to avoid the case of a string with spaces.